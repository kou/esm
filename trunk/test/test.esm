hoge
<% "fuga" %>
<%
(number->string
 1
 )
%>
<%=
(map car
     '((1 2)
       (3 4)
       (5 6)))
%>
<%= (+ 1 2) %>
<%% %%>
<%= "<%% %%>" %>
<%%% %%%>
<%= "<%%% %%%>" %>
<%%%% %%%>%%%%>
<%%%%%>
fufufu
<%; <%% %>
<% (if (> 1 2) (begin %>
<%= 1 %><% ) (begin %>
<%= 2 %>
<% )) %>
<% (if (> 1 2) %>
(> 1 2) is true.
<%   %>
(> 1 2) is not true.
<% ) %>
<%; hohohogee  
hohoho%><% "out" %>
<%= (module-name (current-module)) %>
%%>
