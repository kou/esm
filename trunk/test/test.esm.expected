hoge


(1 3 5)
3
<% %>
<% %>
<%% %%>
<%% %%>
<%%% %%>%%%>
<%%%>
fufufu


2


(> 1 2) is not true.


user
%>
